cmake/FindFuthark.cmake

```diff
@@ -0,0 +1,125 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021-2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# The C language is not enabled in our project by default, but we need it to
+# compile Futhark's code.
+enable_language(C)
+
+# Try to find our dependencies, which are of course Futhark itself (required)
+# and the CUDA toolkit for compiling CUDA projects. Also notify the user of
+# what we find.
+find_program(Futhark futhark REQUIRED)
```

Comment:
Along the same lines as before, if `find_package(Futhark)` is **not** called with `REQUIRED`, then it must not make the configuration fail just because Futhark was not found.

Change this line to:

```cmake
find_program(Futhark_EXECUTABLE futhark)
mark_as_advanced(Futhark_EXECUTABLE)
```

(Variables declared by the module should have a common prefix. Either `FUTHARK_` or `Futhark_`.)